Modix.Data/Models/Core/GuildUserParticipationStatistics.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Modix.Data.Models.Core
+{
+    public class GuildUserParticipationStatistics
+    {
+        public ulong GuildId { get; set; }
+
+        public ulong UserId { get; set; }
+
+        public int Rank { get; set; }
+
+        public decimal AveragePerDay { get; set; }
+
+        public int Percentile { get; set; }
+
+        public string GetParticipationEmoji()
```

Comment:
This strikes me as a view-related concern, embedded in the data layer.

Reply:
Pushed 7111460f2d54e2fd613ae447dcb031da9c3a6e70, which moves this into `UserInfoModule`.